.gitlab-ci.yml

```diff
@@ -59,7 +59,7 @@ stages:
     include:
       - local: '.gitlab/custom-jobs-and-variables.yml'
       - project: 'radiuss/radiuss-shared-ci'
-        ref: v2023.03.1
+        ref: v2023.06.0
```

Comment:
Typo?

```suggestion
        ref: v2023.06.1
```

Reply:
That's the version of the radiuss-shared-ci we use for GitLab testing. It is independent of the RAJA release name.